src/datasets/__init__.py

```diff
@@ -74,7 +74,6 @@
     percent,
 )
 from .utils import *
-from .utils.tqdm_utils import disable_progress_bar
```

Comment:
I think we have to keep this no ?

Reply:
It's not needed anymore because I've added the following import to `datasets.utils.__init__.py`: 
https://github.com/huggingface/datasets/blob/df35e94c11a10ff7ac86bff3e8273fd6665a4c49/src/datasets/utils/__init__.py#L38
which is picked up by `from .utils import *` in `datasets.__init__.py`.

Reply:
Indeed, great !